src/lib/pages/backoffice/EItem/EItemSearch/EItemSearch.js

```diff
@@ -83,8 +84,9 @@ export class EItemSearch extends Component {
               <Container fluid className="spaced">
                 <SearchBar
                   placeholder="Search for eitems..."
-                  queryHelperFields={helperFields}
+                  {...invenioConfig.APP.searchBarRSKProps}
```

Comment:
I don't understand entirely, do we have the query helper in the end or not? It's kinda not intuitive for me to provide the props like this, since I can't easy figure it out for what will they be used

Reply:
In the next [line](https://github.com/inveniosoftware/react-invenio-app-ils/pull/233/files/453978f7eea142005a8454640d7db561e2472437#diff-83f55b7d8a22392ef192ec414c37d4efR89) you can see the QueryBuildHelper as a stand alone component declared. The props you are referring too, are a set of default props for the SearchBar. I added them in the config in order not to repeat them in every `SearchBar` declaration.

```javascript
searchBarRSKProps: {
    autofocus: true,
    actionProps: { icon: 'search', content: null },
    uiProps: {
      className: 'ils-searchbar',
      fluid: true,
      size: 'big',
    },
  },
```

These are going to be used by default and you can customize the SearchBar like this 

```
<SearchBar
  placeholder="Search for eitems..."
  {...invenioConfig.APP.searchBarRSKProps}
  actionProps={{content: "My Search Button Label", icon: null}}
  uiProps={{size: "small"}}
/>
```

that was the most straightforward way I could come up with.